common/winobjc.packagereference.override.targets

```diff
@@ -110,24 +110,6 @@
     </ItemGroup>
   </Target>
 
-  <Import Project="$(MSBuildThisFileDirectory)\workaround\Microsoft.Nuget.Workaround.targets"/>
```

Comment:
I'm a bit worried about this, as well. We don't need `NugetRestore` anymore? For any packages consuming IW?

Reply:
This was a workaround to have each project depend on a single thing to manually nuget restore from. It appears that this workaround isn't needed anymore. When running a regular build with the workaround removed, Visual Studio seems able to restore the nuget packages itself.